### PR TITLE
feat: prompt user to open ledger app on device with single click

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Pushes a prompt to the ledger device to open the correct app, allowing them to confirm and have it open immediately without having to navigate to it on the device. This is the same behavior we see in ledger live.

The user must be on the home page on the device for this prompt to appear.

## Issue (if applicable)

relates to #6814

## Risk

> High Risk PRs Require 2 approvals

Low risk. If the request fails the device ignores the prompt.

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

## Testing

When getting prompted to "Open the XYZ App" on ledger, check the device displays the same notification on the device.

The user must be on the home page on the device for this prompt to appear.

This should work when:
- managing accounts
- verifying addresses
- signing transactions

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]
I can confirm:

- Every chains are asking to open the right app
- If the app is not installed in the device, it doesn't ask anything and it doesn't crash

I couldn't find any bugs while testing this. LGTM!

File: src/components/Modals/LedgerOpenApp/hooks/useLedgerAppDetails.tsx [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
+    case KnownChainIds.BitcoinMainnet:
+      return 'Bitcoin'
+    case KnownChainIds.CosmosMainnet:
+      return 'Cosmos'
+    case KnownChainIds.DogecoinMainnet:
+      return 'Dogecoin'
+    case KnownChainIds.LitecoinMainnet:
+      return 'Litecoin'
+    case KnownChainIds.ThorchainMainnet:
+      return 'THORChain'
```
Comment (gomesalexandre) [inline src/components/Modals/LedgerOpenApp/hooks/useLedgerAppDetails.tsx:38]: Sanity checked against `appNameBySlip44`, LGTM ✅ 

https://github.com/shapeshift/hdwallet/blob/fd985b3d15d25bce20ade5857b9ac03500f50fe3/packages/hdwallet-ledger/src/utils.ts#L593-L603

File: src/components/Modals/LedgerOpenApp/hooks/useLedgerAppDetails.tsx [hunk 2]
```diff
@@ -1,17 +1,54 @@
...
+}
+
 export const useLedgerAppDetails = (chainId: ChainId) => {
   const feeAsset = useAppSelector(state => selectFeeAssetByChainId(state, chainId))
   const ethAsset = useAppSelector(state => selectAssetById(state, ethAssetId))
   const appName = useMemo(() => {
-    if (isEvmChainId(chainId)) return ethAsset?.networkName
-    return feeAsset?.networkName
-  }, [feeAsset?.networkName, chainId, ethAsset?.networkName])
+    return getLedgerAppName(chainId)
```
Comment (gomesalexandre) [inline src/components/Modals/LedgerOpenApp/hooks/useLedgerAppDetails.tsx:50]: 💜

**Review by gomesalexandre — APPROVED** [review]
Tested locally, does what it says on the box 🐐 

- Opening the modal with unlocked Ledger on the home screen prompts to open app
- Opening the modal with locked ledger, unlocking it to home screen doesn't prompt to open app, but detection is still happy after manually opening it
- Opening the modal with the right app opened is still happy

**Pull request merged**